derp/derp_client.go

```diff
@@ -220,15 +210,16 @@ func (c *Client) send(dstKey key.Public, pkt []byte) (ret error) {
 	c.wmu.Lock()
 	defer c.wmu.Unlock()
 	if c.rate != nil {
-		pktLen := frameHeaderLen + len(dstKey) + len(pkt)
+		pktLen := frameHeaderLen + 32 + len(pkt)
```

Comment:
Can we add an exported const to package key? Then this can read `+ key.NodeLen +` or the like. Or perhaps better, a Len method, so it reads `+ dstKey.Len() +`.

Reply:
I keep going back and forth about this, because this length is only relevant for one particular serialization, and it's not even the most common serialization in our codebase. In my happy place, nobody would need to care how big keys are.

As a compromise to get rid of magic numbers, I added `RawLen` to NodePublic, and renamed ReadRaw32... and WriteRaw32... to just ReadRaw... and WriteRaw..., to try and convey that this is one family of functions that deals with one particular serialization of NodePublic. WDYT?

Reply:
SGTM